docs/faq.md

```diff
@@ -3,64 +3,61 @@ id: faq
+Most probably, the server is behind NAT, but you haven't added the NAT-specific configuration. See this [resolved question](https://community.jitsi.org/t/cannot-see-video-or-hear-audio-on-self-hosted-instance/). You need to follow the steps detailed [here](devops-guide/devops-guide-quickstart#advanced-configuration).
 
-**3. It works with two participants, but crashes or does not work properly when a third joins**
+## It works with two participants, but crashes or does not work properly when a third joins
 
-p2p mode is working, but it fails when you are trying to pass traffic via jitsi-videobridge2.
+P2P mode is working, but it fails when you are trying to pass traffic via jitsi-videobridge2.
 
-Check you've got your firewall / NAT set up correctly — especially UDP 10000.
+Check you've got your firewall / NAT set up correctly — especially UDP 10000. For more information, see [here](devops-guide/devops-guide-quickstart#setup-and-configure-your-firewall).
 
-See [here](https://github.com/jitsi/jitsi-meet/blob/master/doc/quick-install.md#open-ports-in-your-firewall)
+## Can I mute and unmute other participants?
 
-**4. Can I mute and unmute other participants?**
-
-If you are the moderator of a conference (everyone is a moderator if you are using meet.jit.si), you can mute everyone's microphone. You cannot unmute other people's microphones, and they can unmute their microphone at any time.
+If you are the moderator of a conference (everyone is a moderator if you are using `meet.jit.si`), you can mute everyone's microphone. You cannot unmute other people's microphones, and they can unmute their microphone at any time.
 
 You may want to set some "ground rules" for who can talk and when, just as with any physical meeting or classroom.
 
-**5. How can I protect my meetings with jitsi?**
+If you would like to limit who can become a moderator, you need to set up your own instance of Jitsi and enable "secure domain". Please see [here](#_4-enable-secure-domain-if-you-are-using-your-own-instance-of-jitsi_) for more information.
 
-If you are using the hosted meet.jit.si platform:
+## How can I protect my meetings with Jitsi?
 
-_1.) Create a "strong" room name._
+### _1. Create a "strong" room name._
 
 Use a strong room name, which no-one else is likely to be using. Use the name generator on the welcome page, or else generate your own "strong" name.
 
 For example, on macOS, in terminal, you can use `uuidgen` to generate a string of letters of numbers (e.g. B741B63E-C5E6-4D82-BAC4-048BE25D8CC7).
 
-Your room name would be meet.jit.si/B741B63E-C5E6-4D82-BAC4-048BE25D8CC7
+Your room name would be `meet.jit.si/B741B63E-C5E6-4D82-BAC4-048BE25D8CC7` on the hosted `meet.jit.si` platform.
```

Comment:
I moved `If you are using the hosted meet.jit.si platform:` to here since these tips are also valid for self-hosted instance.